accounts/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin, UserManager
+from django.contrib.auth.validators import ASCIIUsernameValidator
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth.base_user import BaseUserManager,AbstractBaseUser
+
+class UserManager(BaseUserManager):
+    use_in_migratons = True
+    def create_user(self,username,password,**extra_fields):
+        if not username:
+            raise ValueError('The given username must be set')
+        extra_fields.setdefault('is_staff',False)
+        extra_fields.setdefault('is_superuser',False)
+        user = self.model(username=username,**extra_fields)
+        user.set_password(password)
+        user.save(using = self._db)
+        return user
+    def create_superuser(self,username, password=None, **extra_fields):
+        if not username:
+            raise ValueError('The given username must be set')
+        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_superuser', True)
+        if extra_fields.get('is_staff') is not True:
+            raise ValueError('Superuser must have is_staff=True.')
+        if extra_fields.get('is_superuser') is not True:
+            raise ValueError('Superuser must have is_superuser=True.')
+        user = self.model(username=username,**extra_fields)
+        user.set_password(password)
+        user.save(using = self._db)
+        return user
+  
+class User(AbstractBaseUser,PermissionsMixin):
+    username_validater=ASCIIUsernameValidator()
+    username=models.CharField(
```

Comment:
```suggestion
class User(AbstractBaseUser, PermissionsMixin):
    username_validater = ASCIIUsernameValidator()
    username = models.CharField(
```
細かいですが、引数同士はスペース開けてください。また、=などの２項演算子も同様にスペースを空けてください。ただし、キーワード引数の時は開けなくていいです。これに関してもPEP8基準の[こちら](https://pep8-ja.readthedocs.io/ja/latest/#section-14)に詳しい記載があるので確認してください。